api/cleanup_copy_chunk_operation_experimental.md

```diff
@@ -0,0 +1,40 @@
+## cleanup_copy_chunk_operation() <tag type="community">Community</tag> <tag type="experimental">Experimental</tag>
+TimescaleDB allows you to [copy][copy_chunk] or [move][move_chunk] a
+chunk to a new location within a multi-node environment. Because the
+operation happens over multiple transactions, it has to be manually
+cleaned up using this function in case of failure. Without cleanup,
+the failed operation will hold a replication slot open and eventually
```

Comment:
depending on the stage of the failure, the repl slot MIGHT or MIGHT NOT be held.